lab-abigail/lib/hex.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(onePath, twoPath, threePath, callback) {
+
+  let textArray = [];
+
+  fs.readFile(onePath, function(err, data) {
+    if(err) throw err;
+    let oneData = data.toString('hex', 0, 8);
+    textArray.push(oneData);
+    console.log(`First File: ${oneData}`);
+
+    fs.readFile(twoPath, function(err, data) {
+      if(err) throw err;
+      textArray.push(data.toString('hex', 0, 8));
+      console.log(`Second File: ${data.toString('hex', 0, 8)}`);
```

Comment:
``console.logs`` in your ``index.js`` module are fine, but make sure to get rid of them in other modules (including test module) before submitting. We'll start grading down for that next week.